auth/src/main/java/com/hms/lib/commonmobileservices/auth/google/GoogleAuthServiceImpl.kt

```diff
@@ -25,12 +28,56 @@ import com.hms.lib.commonmobileservices.auth.common.VerificationType
 import com.hms.lib.commonmobileservices.auth.exception.ExceptionUtil
 import com.hms.lib.commonmobileservices.core.Work
 import java.util.*
+import java.util.concurrent.TimeUnit
 
 
-class GoogleAuthServiceImpl : AuthService {
+class GoogleAuthServiceImpl(private val context: Context) : AuthService {
 
     private val firebaseAuth: FirebaseAuth = FirebaseAuth.getInstance()
     private val mapper: Mapper<FirebaseUser, AuthUser> = FirebaseUserMapper()
+    private lateinit var storedVerificationId: String
+    private lateinit var resendToken: PhoneAuthProvider.ForceResendingToken
+    lateinit var callbacks: PhoneAuthProvider.OnVerificationStateChangedCallbacks
+
+    companion object {
+        private const val CMS_SHARED_PREF = "CMS_Shared_Pref"
+        private const val VERIFICATION_ID = "verificationId"
+    }
+
+
+    init {
+
+
+        callbacks = object : PhoneAuthProvider.OnVerificationStateChangedCallbacks() {
+            override fun onVerificationCompleted(p0: PhoneAuthCredential) {
+                TODO("Not yet implemented")
```

Comment:
Please complete to the before merge. It can be caused crash

Reply:
resolved.